lib/modules/datasource/kubernetes-api/mapping.ts

```diff
@@ -0,0 +1,118 @@
+export const KubernetesApiVersions: Record<string, string[]> = {
```

Comment:
should we move this to a json file, so we can some tea in future update this automatically?

Reply:
I can do, but I've not been able to identify a machine readable source for this information. The deprecation guide is to free form to write a parser for.

Reply:
Moved this to `data/kubernetes-api.json5`, but still unable to identify a machine readable source.